static_src/components/space_container.jsx

```diff
@@ -101,8 +101,7 @@ export default class SpaceContainer extends React.Component {
 
           <AppList />
           <PanelActions>
-            <span>Learn how to <a href={ config.docs.deploying_apps }>deploy a
-            new app</a>.</span>
+            <InfoAppCreate space={ space } org={ this.state.currentOrg } />
```

Comment:
This technically shouldn't be in panel actions. I think I changed it on my branch.